node/src/blockchain/blockchain_interface.rs

```diff
@@ -900,7 +877,8 @@ mod tests {
             2u64,
         );
 
-        transport.assert_request("eth_sendRawTransaction", &[String::from(r#""0xf8a801847735940082dbe894384dec25e03f94931767ce4c3556168468ba24c380b844a9059cbb00000000000000000000000000000000000000000000000000626c61683132330000000000000000000000000000000000000000000000000000082f79cd900029a0b8e83e714af8bf1685b496912ee4aeff7007ba0f4c29ae50f513bc71ce6a18f4a06a923088306b4ee9cbfcdc62c9b396385f9b1c380134bf046d6c9ae47dea6578""#)]);
+        //TODO remove this comment: to my defence in the review, I replaced these strings because I believe that chain_id takes part in the signing process and because of changing the number the sign must look different now
```

Comment:
Yes, the network ID is required to be included in the transaction and because the transaction is signed the payload changes if the network changes. The transaction is not valid and will be rejected if the network ID is not correct.